core/config/engine.h

```diff
@@ -85,6 +85,7 @@ class Engine {
 	HashMap<StringName, Object *> singleton_ptrs;
 
 	bool editor_hint = false;
+	bool tests_hint = false;
```

Comment:
Can you open a new PR with only the `core/` and `editor/` changes, which can be merged as a prerequisite of this PR?